sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -69,11 +69,12 @@ const (
 	URNWindowMappingFixed   = "beam:go:windowmapping:fixed:v1"
 	URNWindowMappingSliding = "beam:go:windowmapping:sliding:v1"
 
-	URNProgressReporting     = "beam:protocol:progress_reporting:v1"
-	URNMultiCore             = "beam:protocol:multi_core_bundle_processing:v1"
-	URNWorkerStatus          = "beam:protocol:worker_status:v1"
-	URNMonitoringInfoShortID = "beam:protocol:monitoring_info_short_ids:v1"
-	URNDataSampling          = "beam:protocol:data_sampling:v1"
+	URNProgressReporting          = "beam:protocol:progress_reporting:v1"
+	URNMultiCore                  = "beam:protocol:multi_core_bundle_processing:v1"
+	URNWorkerStatus               = "beam:protocol:worker_status:v1"
+	URNMonitoringInfoShortID      = "beam:protocol:monitoring_info_short_ids:v1"
+	URNDataSampling               = "beam:protocol:data_sampling:v1"
+	URNSDKDataChannelStatusSignal = "beam:protocol:sdk_data_channel_status_signal:v1"
```

Comment:
Don't forget to add this constant to the `goCapabilities` list down below. It's not magic, it's just defined here, and used down there.

Reply:
Done. Thanks!